fix(cli): report broken symlinks as failures instead of aborting the run (#508)
Boeing/config-file-validator: CHANGELOG.md

### Fixed

- TOML files with duplicate keys are now rejected as invalid (closes #504).

> - A broken symlink in a scanned directory no longer aborts the entire run; it is now reported as a failed file (closes #505).

Reviewer: ```suggestion
- Broken symlinks are reported as validation failures instead of aborting the run (closes #505)
```
